assets/src/edit-story/app/story/effects/useLoadStory.js

```diff
@@ -9,6 +14,10 @@ import { useEffect } from '@wordpress/element';
 import { useAPI, useHistory } from '../../';
 import { createPage } from '../../../elements';
 
+const getPerm = ( post, field ) => {
```

Comment:
Could we write the name of this func out for clarity or add a comment?

Reply:
Added comment here https://github.com/ampproject/amp-wp/pull/4122/commits/6e1e71e9397d71efad6685a01ef9111648c0d876